apps/daimo-mobile/package.json

```diff
@@ -66,6 +66,7 @@
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.0",
     "react-native-svg": "13.9.0",
+    "react-native-swipe-up-down": "github:react-native-vietnam/react-native-swipe-up-down",
```

Comment:
is this different from the version on NPM?

Reply:
pros: looks like a simple component https://www.npmjs.com/package/react-native-swipe-up-down?activeTab=code

cons: ~200 downloads a month, almost nobody is using this

Reply:
have you tried https://www.npmjs.com/package/@gorhom/bottom-sheet

Reply:
i was trying a couple of thing but now is installed from npm normally. We can discuss about using another package if you want.